### PR TITLE
chore(styleGuide): add a new topic on OTEL examples 

### PR DESCRIPTION
**Pull request by ghost**
This PR adds a new style guide doc about our team's philosophy about OTEL examples moving forward. 
Jira: NR-287533

### CODE REVIEW

**Comment by ally-sassman** [comment]
@paperclypse Let me know if this page has a more appropriate home in the style guide yml (and what the path name should be). I took a guess!

File: src/content/docs/style-guide/writing-strategies/Untitled-1 [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+This page outlines the docs team's approach to documenting OpenTelemetry examples.
+
+## Background [#background]
+
+In New Relic documentation for OpenTelemetry (OTEL), we provide examples aimed to demonstrate the most common configurations we expect users to use. Historically, the docs site contained these examples on the docs site. However, in May 2024, a joint effort with Relics working on OTEL, we decided to migrate these examples from the docs site to the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main) repo. For example, here’s a PR where we moved all the infrastructure examples: [PR #17742](https://github.com/newrelic/docs-website/pull/17742).
+
+## Why we support this change [#why-we-support-this-change]
+
+The reasons for this migration are:
+* Reduce maintenance burden: Both the OTEL and docs team are small, with a lot of surface area. We have to keep our processes lean to stay afloat.
```
Comment (paperclypse) [inline src/content/docs/style-guide/writing-strategies/Untitled-1:15]: I'd drop this sentence: "We have to keep our processes lean to stay afloat."

File: src/content/docs/style-guide/writing-strategies/Untitled-1 [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: OpenTelemetry examples
+metaDescription: "Our stance on OpenTelemetry examples moving forward."
+---
+
+This page outlines the docs team's approach to documenting OpenTelemetry examples.
+
+## Background [#background]
+
+In New Relic documentation for OpenTelemetry (OTEL), we provide examples aimed to demonstrate the most common configurations we expect users to use. Historically, the docs site contained these examples on the docs site. However, in May 2024, a joint effort with Relics working on OTEL, we decided to migrate these examples from the docs site to the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main) repo. For example, here’s a PR where we moved all the infrastructure examples: [PR #17742](https://github.com/newrelic/docs-website/pull/17742).
```
Comment (paperclypse) [inline src/content/docs/style-guide/writing-strategies/Untitled-1:10]: Even though the style guide is quasi-internal, I think it's best to not use more internal terms like "Relics". How about "a joint effort between the docs and OTel team..."

Also, is OTEL always all caps like that?

File: src/content/docs/style-guide/writing-strategies/Untitled-1 [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Why we support this change [#why-we-support-this-change]
+
+The reasons for this migration are:
+* Reduce maintenance burden: Both the OTEL and docs team are small, with a lot of surface area. We have to keep our processes lean to stay afloat.
+* Reduce redundancy across both the repo and docs site: Keeping multiple copies encourages (and almost guarantees) diverging sources of truth and bit rot.
+* Make it easier to consume the examples: Writing examples with text, like “Put X content in a file call Y”, is a bad developer experience since it's time consuming and error prone. It's much better to clone a repo, update an environment variable or two, and run a command.
```
Comment (paperclypse) [inline src/content/docs/style-guide/writing-strategies/Untitled-1:17]: I'd move the these last two bullets to the top and move "reduce maintenance burden" to the bottom of the list

File: src/content/docs/style-guide/writing-strategies/Untitled-1 [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+* Emphasis on working code examples for use internally and externally: Internally, the data from these examples are the source of truth on top of which we build product experiences. Externally, these examples are the source of truth on how you do a particular thing with New Relic and OpenTelemetry (stray away from this path and results may vary).
+
+## How we handle requests for future OTEL examples [#how-we-handle-requests]
+
+Going forward, all OTEL examples will be created and maintained in the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main) repo, with a reference mentioned on the docs site.
```
Comment (paperclypse) [inline src/content/docs/style-guide/writing-strategies/Untitled-1:22]: "examples will be created" --> "examples are created"

**Review by paperclypse — CHANGES_REQUESTED** [review]
Looks like you've got two duplicate copies of the doc in this PR. Definitely delete one of them. I left all my comments in the untitled one before I realized.

**Review by paperclypse — APPROVED** [review]
Looks good, thanks for making the edits!

**Pull request merged**